src/pj_proj_test.c

```diff
@@ -0,0 +1,39 @@
+/* Tiny test of an evolving new API. Thomas Knudsen, 2016-10-30 */
```

Comment:
Since this file is not really a part of the main library I think it would be better to move it to either the ```test``` folder or a new ```examples```. This way it is more clear what is what.